### ISSUE TITLE
test_timeleyProcessExited mispelled

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5679]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5679|
|Type|defect|
|Created|2012-05-15 12:02:57Z|
|Branch|https://github.com/twisted/twisted/tree/test-typo-5679|

"Timely", not "timeley".

Attachments:

* [5679.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/176/1765fcdb53929db2c88d35300ddc2622ddaf9bc7/14bee17470c1bc5a2dd66b66782ecf6a2056726f.patch) (794 bytes) - added by moijes12 on 2012-06-12 16:12:27Z - Done and newsfile attached. 

<details><summary>Searchable metadata</summary>

```
trac-id__5679 5679
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_test_typo_5679 branches/test-typo-5679
branch_author__therve therve
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1337083377000000 1337083377000000
changetime__1339946150000000 1339946150000000
version__None None
owner__therve therve
cc__moijes12
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="moijes12's avatar" src="https://avatars.githubusercontent.com/u/4566851?s=50" width="50" height="50">](https://github.com/moijes12)|@moijes12 set owner to @moijes12|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="moijes12's avatar" src="https://avatars.githubusercontent.com/u/4566851?s=50" width="50" height="50">](https://github.com/moijes12)|@moijes12 removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve commented|
|-|-|

(In [34569]) Branching to 'test-typo-5679'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @therve|
|-|-|

Looks good!

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve set status to `closed`|
|-|-|

(In [34571]) Merge test-typo-5679

Author: moijes12
Reviewer: therve
Fixes: [#5679](https://github.com/twisted/twisted/issues/5679)

Fix typo in test method name.
